perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/RegistrarManagerMethod.java

```diff
@@ -452,14 +452,33 @@ public ApplicationFormItem call(ApiCaller ac, Deserializer parms) throws PerunEx
 	 * @param state List<String> List of states: NEW, VERIFIED, APPROVED, REJECTED
 	 * @return List<Application> Found applications
 	 */
+	/*#
+	 * Gets all applications in a given state for a given VO in a given date period.
+	 *
+	 * @param vo int VO <code>id</code>
+	 * @param state List<String> List of states: NEW, VERIFIED, APPROVED, REJECTED
+	 * @param dateFrom String Earliest date for applications
+	 * @param dateTo String Latest date for applications
+	 * @return List<Application> Found applications
+	 */
 	getApplicationsForVo {
 
 		@Override
 		public List<Application> call(ApiCaller ac, Deserializer parms) throws PerunException {
-			if (parms.contains("state")) {
-				return ac.getRegistrarManager().getApplicationsForVo(ac.getSession(), ac.getVoById(parms.readInt("vo")), parms.readList("state", String.class));
+			if(parms.contains("dateFrom")) {
```

Comment:
I would suggest the same changes as for the getApplicationsForGroup method here.

Reply:
OK